README.md

```diff
@@ -115,22 +120,26 @@ Usage
 - **database** as *string*, **required**
 
   SQLite database path.
+  
+- **updatable** as *boolean*, optional, default *true*
 
-- **truncatable** as *boolean*, optional
+  This option allow or disallow write operations on SQLite database file.
+    
+- **truncatable** as *boolean*, optional, default *false*
```

Comment:
Default value of truncatable option is true, not false.

Reply:
Thanks, fixed.

Reply:
Thanks for fixing. I confirmed.